app/controllers/ldp/ReadWriteWebControllerGeneric.scala

```diff
@@ -79,7 +79,7 @@ trait ReadWriteWebControllerGeneric extends ReadWriteWebControllerTrait {
   private def allowHeaders(authResult: AuthResult[NamedResource[Rdf]]): List[(String, String)] = {
     val modesAllowed = authResult.authInfo.modesAllowed
     val isLDPC =  authResult.result.isInstanceOf[LocalLDPC[_]]
-    val allow = "Allow" -> {
+    val allow = "Access-Control-Allow-Methods" -> {
```

Comment:
Alright, I hadn't found that. I'm wondering why they didn't use the same header. Can there be cases where the two headers have different values?
Otherwise I'm just going to duplicate it.